grails-app/services/org/broadinstitute/orsp/TransitionService.groovy

```diff
@@ -96,16 +97,40 @@ class TransitionService {
      * NE/NHSR projects go into Submitting to ORSP
      */
     @Transactional
-    handleIntake(Issue issue, Collection<String> actors, String status, String author) throws DomainException {
-        def deletableProperties = [IssueExtraProperty.ACTOR]
-        deleteProps(issue, deletableProperties)
+    handleIntake(Issue issue, Collection<String> actors) throws DomainException {
```

Comment:
Just a note that this class is heavily documented for the express purpose of giving developers a clear overview of that statuses are required for different project types.